sdcm/utils/sstable_utils.py

```diff
@@ -0,0 +1,134 @@
+import datetime
+import json
+import logging
+import random
+
+from sdcm.cluster import BaseScyllaCluster, BaseCluster, BaseNode
+
+
+class NonDeletedTombstonesFound(Exception):
+    pass
+
+
+class SstableUtils:
```

Comment:
Please write docs for this class - what's the purpose etc.
Also mark private methods with underscore, so reuse this class would be clearer.

Reply:
fixed.